docs/pages/button.js

```diff
@@ -259,245 +229,481 @@ card(
+        defaultCode={`
+<Flex alignContent="center">
+  <ButtonGroup>
+    <Button text="Visit" size="lg" color="red" />
+    <Button text="Save" size="lg" color="red" />
+  </ButtonGroup>
+</Flex>
+`}
+      />
 
-  return (
-    <Flex alignItems="start" direction="column" gap={6}>
-      <Flex gap={2}>
-        <Tooltip text="Default button">
-          <Button
-            onClick={() => {}}
-            text="Clear search history"
-            disabled={disabled}
-            tabIndex={tabIndex ? -1 : 0}
-          />
-        </Tooltip>
-        <Tooltip text="Submit button">
-          <Button
-            type="submit"
-            name="satisfaction-questionnaire"
-            text="Submit your response"
-            disabled={disabled}
-            tabIndex={tabIndex ? -1 : 0}
-          />
-        </Tooltip>
-        <Tooltip text="Link button">
-          <Button
-            role="link"
-            target="blank"
-            href="https://www.pinterest.com"
-            text="Visit pinterest.com"
-            disabled={disabled}
-            tabIndex={tabIndex ? -1 : 0}
-          />
-        </Tooltip>
-      </Flex>
-      <Flex gap={2}>
-        <Switch
-          onChange={() => setDisabled(!disabled)}
-          id="disable-buttons"
-          switched={disabled}
-        />
-        <Box paddingX={2} flex="grow">
-          <Label htmlFor="disable-buttons">
-            <Text>Disable buttons</Text>
-          </Label>
-        </Box>
-      </Flex>
-      <Flex gap={2}>
-        <Switch
-          onChange={() => setTabIndex(!tabIndex)}
-          id="unreachable-buttons"
-          switched={tabIndex}
-        />
-        <Box paddingX={2} flex="grow">
-          <Label htmlFor="unreachable-buttons">
-            <Text>Remove from keyboard navigation with tabIndex</Text>
-          </Label>
-        </Box>
-      </Flex>
-    </Flex>
-  );
-}
+      <MainSection.Card
+        cardSize="md"
+        type="do"
+        description="Show the full text on Buttons. Buttons should be stacked when they cannot be displayed side by side."
+        defaultCode={`
+<Flex gap={2} direction="column" alignContent="stretch">
+  <Button text="Create account" size="lg" color="red" fullWidth />
+  <Button text="View settings" size="lg" color="gray" fullWidth />
+</Flex>
 `}
-  />,
+      />
+      <MainSection.Card
+        cardSize="md"
+        type="don't"
+        description="Truncate the Button text. In rare instances where Buttons must remain on one line, truncate the text on the secondary Button before truncating on the primary Button."
+        defaultCode={`
+<Flex gap={2}>
+  <Button text="Kontoeinst..." size="lg" color="gray" />
+  <Button text="Neues We..." size="lg" color="red" />
+</Flex>
+`}
+      />
+      <MainSection.Card
+        cardSize="md"
+        type="do"
+        description="Keep the Button text as simple and actionable as possible. Refer to the [Button writing guidelines](#Writing) for more detail."
```

Comment:
I'd also refer to a11y guidelines, if text is not enough for a11y and needs label.

Reply:
Done!